services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/LinkOperationsBolt.java

```diff
@@ -41,17 +47,29 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
 public class LinkOperationsBolt extends PersistenceOperationsBolt {
+    private transient LinkOperationsService linkOperationsService;
+
     public LinkOperationsBolt(PersistenceManager persistenceManager) {
         super(persistenceManager);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
+        super.prepare(stormConf, context, collector);
+        this.linkOperationsService = new LinkOperationsService(repositoryFactory, transactionManager);
+    }
+
     @Override
     @SuppressWarnings("unchecked")
-    List<InfoData> processRequest(Tuple tuple, BaseRequest request) {
+    List<InfoData> processRequest(Tuple tuple, BaseRequest request) throws AbstractException {
```

Comment:
Maybe it's better to throws IslNotFoundException? To be sure that we will handle it

Reply:
+1